media_driver/linux/common/cp/ddi/media_ddi_prot.cpp

```diff
@@ -350,7 +499,77 @@ VAStatus DdiMediaProtected::ProtectedSessionCreateBuffer(
     void                    *data,
     VABufferID              *bufId)
 {
-    return VA_STATUS_ERROR_UNIMPLEMENTED;
+    PDDI_MEDIA_CONTEXT             pMediaCtx            = nullptr;
```

Comment:
Per https://github.com/intel/libva/pull/457/files, we don't execute vaProtectedSessionHwUpdate in the API and I don't see that this function is being called anywhere in the library. It seems this function is simply not needed anymore. Hence, why add implementation?